MastodonSDK/Sources/MastodonCore/Extension/CoreDataStack/StatusEdit+Property.swift

```diff
@@ -0,0 +1,16 @@
+// Copyright © 2023 Mastodon gGmbH. All rights reserved.
+
+import Foundation
+import CoreDataStack
+import MastodonSDK
+
+extension StatusEdit.Property {
+    init(entity: Mastodon.Entity.StatusEdit) {
+        self.init(
+            createdAt: entity.createdAt,
+            content: entity.content,
+            sensitive: entity.sensitive,
+            spoilerText: entity.spoilerText,
+            emojis: entity.mastodonEmojis)
```

Comment:
Emojis are there, but I can't explain, why `poll` and `mediaAttachements` are missing here, at least according to [StatusEdit](https://docs.joinmastodon.org/entities/StatusEdit/) they should be there? Am I missing something?